styles/layouts/SectionHeader.module.scss

```diff
@@ -0,0 +1,42 @@
+.sectionHeader {
+  @apply w-full;
+  @apply pt-4 px-4;
+  @apply flex;
+}
+
+.sectionHeaderContainer {
+  @apply rounded-t-lg;
+  @apply px-4 py-2;
+  @apply shrink;
+  @apply flex-grow-0;
+}
+
+.sectionHeaderTitle {
```

Comment:
why a `title` class again? wouldn't this be better as a `h` or some sort and we just style the h directly?